decidim-admin/lib/decidim/admin/test/manage_hide_content_examples.rb

```diff
@@ -57,7 +57,7 @@
 
       visit decidim_admin.root_path
       expect(page).to have_content("blocked user")
-      expect(page).to have_content("hid a resource of type")
+      expect(page).to have_no_content("hid a resource of type")
```

Comment:
I am not very sure that i can follow why you change the spec from "expect to have text " into "expect not to have text"

Reply:
It was because it didn't show this message.